### DISCUSSION TITLE
refused to connect Internal Grafana with Lets Encrypt SSL Cert

### DISCUSSION THREAD

**Discussion by matze1708** [guysoft/FullPageOS#565] [Q&A]
#### What were you doing?

I would like to call an internal page to our Grafana. 

#### What did you expect to happen?

Show Grafana Site

#### What happened instead?

I get an Error refused to connect

#### Was there an error message displayed? What did it say?

grafana.intranet.XXXXXX.de refused to connect.
Site is only availablefrom our intranet.

#### Version of FullPageOS?
0.12.0
[Can be found in /etc/fullpageos_version ALWAYS INCLUDE.]

**Comment by guysoft** [comment]
Does it connect if you open a browser window using ctrl+T from FullPageOS?

**Comment by matze1708** [comment]
Hello thanks for the answer
Yes, it works through this browser / access

it only loads very long, but the dashboard is also large

**Comment by chadmccune** [comment]
I'm getting the same issue, but its with every url when trying to use the dashboard to rotate.  nothing works, even https://www.google.com

The error it shows is "refused to connect"

To clarify, it does work when using ctrl-t and trying there.  It also works if I don't use the /FullPageDashboard app

**Comment by guysoft** [comment]
It might be cross-site https request.

**Comment by anduz** [comment]
Hi, I'm having this problem as well.
I'm using Tampermonkey addon for autologin. This works in Chrome (Ctrl+t).
After closing this windows it worked in Fullpage as well
After reboot I'm getting the refused connection again.

If i open Chrome (Ctrl+t) again an login, then refresh the page, it shows the page.

Now it seems to work .. what the cause of the "refused connection"

**Comment by anduz** [comment]
It is NOT working. After some time I get the error again.
Rebooting does not solve the problem, only after when I open another tab, the fullpage tab works after refreshing

**Comment by guysoft** [comment]
@anduz Are you using grafana too? What userscript?

Also I think I am converting this issue in to a discussion, it does not seem to be a bug, just a specific usecase (one that a few years ago I did too)

**Reply by anduz** [reply]
No, I'm accessing this site: https://console.lynus.io/#/
